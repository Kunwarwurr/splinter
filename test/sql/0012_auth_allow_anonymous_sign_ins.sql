begin;

  -- 0 issues
  select * from lint."0012_auth_allow_anonymous_sign_ins";
  
rollback;
