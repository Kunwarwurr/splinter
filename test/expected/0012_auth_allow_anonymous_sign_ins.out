begin;
  -- 0 issues
  select * from lint."0012_auth_allow_anonymous_sign_ins";
 name | level | facing | description | remediation | detail | metadata | cache_key 
------+-------+--------+-------------+-------------+--------+----------+-----------
(0 rows)

  
rollback;
